plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/changedList/ChangedListViewImpl.ui.xml

```diff
@@ -14,7 +14,14 @@
 <ui:UiBinder xmlns:ui='urn:ui:com.google.gwt.uibinder'
              xmlns:g='urn:import:com.google.gwt.user.client.ui'>
     <ui:with field='res' type='org.eclipse.che.ide.ext.git.client.GitResources'/>
-    <g:DockLayoutPanel ui:field="changedFilesPanel" unit="PX" width="800px" height="500px"
-                       addStyleNames="{res.gitCSS.simpleListContainer}"
-                       debugId="git-changed-filesPanel"/>
-</ui:UiBinder>
\ No newline at end of file
+    <ui:with field='locale' type='org.eclipse.che.ide.ext.git.client.GitLocalizationConstant'/>
+
+    <g:DockLayoutPanel unit="PX" width="800px" height="500px">
```

Comment:
`ChangedListViewImplUiBinder` uses `Widget`, but here you use DockLayoutPanel. Use one type everywhere.